tests/core/test_base_model.py

```diff
@@ -1,95 +1,350 @@
-"""Test module for base_model.py.
+"""Test module for model.py (and associated functionality).
 
-This module tests the functionality of base_model.py, which defines the basic model API
-that specific models (e.g. `soil_model.py`) utilise.
+This module tests the functionality of model.py, as well as other bits of code that
```

Comment:
It doesn't actually test other bits of code anymore as this has been moved to separate scripts